ui_graphics.adoc

```diff
@@ -256,10 +256,10 @@ image::images/des_colour_pal.png[des_colour_pal]
 The *_Eclipse-style palette_* contains the core and dominant colors
   used in Eclipse-based icons, wizard banner graphics, and user
   assistance graphics. You can download the palette in the
-  https://github.com/eclipse-platform/eclipse.platform.images/tree/master/org.eclipse.images/tools/eclipse-style_palette.aco[".aco"],
-  https://github.com/eclipse-platform/eclipse.platform.images/tree/master/org.eclipse.images/tools/tools/eclipse-style_palette.ai[".ai"]
+  link:media/eclipse-style_palette.aco?raw=true[".aco"]
```

Comment:
This should work, but the raw parameter is not necessary. This is finally served from github.io, not from the git repo. Can you remove it to avoid confusion?

Reply:
> Can you remove it to avoid confusion?

Sure.